scripts/jenkins/cloud/manual/lib.sh

```diff
@@ -44,6 +44,11 @@ function setup_ansible_venv {
 }
 
 function mitogen_enable {
+    if [[ "${NO_MITOGEN:+true}" == "true" ]]; then
```

Comment:
Does this translate into disabling mitogen by default ? If yes, was this your intention ?

Reply:
This will only disable it if you have defined the NO_MITOGEN variable in your environment. As per the bash manpage:
```
       ${parameter:+word}
              Use Alternate Value.  If parameter is null or unset, nothing is substituted, otherwise the expansion of word is substituted.
```

So this will make no net change to how things run unless you specify NO_MITOGEN... If we want to change things so mitogen is disabled by default, I'm happy to... But mitogen does provide some performance benefits, especially for remote ansible hosts.

I envisage this change to be more along the lines of
(a) a debugging aid to allow testing without mitogen to determine "weirdness" like I saw was related to mitogen or not, or
(b) allowing someone working on a "local" node to disable mitogen if they don't see much benefit to it, or
(c) want to run exactly like the CI would, even if that will be slow when running remotely.

Without this change the person would have to make a local change to their deploy-ardana.sh (or whatever) script to remove the mitogen_enable, or would need to write their own script.

Reply:
ok :+1: